website/docs/r/databricks_workspace.html.markdown

```diff
@@ -122,6 +124,26 @@ A `custom_parameters` block supports the following:
 
 ~> **Note:** Databricks requires that a network security group is associated with the `public` and `private` subnets when a `virtual_network_id` has been defined. Both `public` and `private` subnets must be delegated to `Microsoft.Databricks/workspaces`. For more information about subnet delegation see the [product documentation](https://docs.microsoft.com/azure/virtual-network/subnet-delegation-overview).
 
+---
+
+An `enhanced_security_compliance` block supports the following:
+
+* `automatic_cluster_update_enabled` - (Optional) Enables automatic cluster updates for this workspace.
+
+* `compliance_security_profile_enabled` - (Optional) Enables compliance security profile for this workspace.
```

Comment:
These should have 
```
Defaults to `x`.
```
information on them

Reply:
Fixed in https://github.com/hashicorp/terraform-provider-azurerm/pull/26606/commits/a936b91ade2df630ccbf46da0e31f49de749bf34